models/gardenplant.js

```diff
@@ -11,14 +11,19 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      GardenPlant.belongsTo(models.Garden);
+      GardenPlant.belongsTo(models.Plant);
```

Comment:
Here we see the data model ASSOCIATIONS defined between Garden/GardenPlant/Plant.